less/_components/events/eclipsefdn-registration.less

```diff
@@ -18,6 +18,9 @@
   p{
     margin-bottom:20px;
   }
+  .eclipsefdn-registration-links li {
```

Comment:
Does this need to be inside of .eclipsefdn-registration for this to work?

Reply:
No need to. But since it belongs to this class, I thought I should also put in there. Seems not needed to put in there?

Reply:
It's better to have this out of that class since it's will be easier for us to override if needed.